src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/arithmetic.txt

```diff
@@ -119,4 +119,29 @@ Error({Kind:ErrorKind.Div0})
 Error({Kind:ErrorKind.Div0})
 
 >> 5 <> 1/0
-Error({Kind:ErrorKind.Div0})
\ No newline at end of file
+Error({Kind:ErrorKind.Div0})
+
+// Blank coercions
```

Comment:
>Blank coercions [](http://example.com/codeflow?start=3&length=15)

Should these have explicit tests for Blank()?

Reply:
I don't think it's necessary. Blank() by itself isn't very interesting - we would typically use it in an expression that has explicit type. But I added those here as well, txt tests are pretty fast to run.